packages/salesforcedx-vscode-apex/src/commands/apexActionController.ts

```diff
@@ -35,6 +36,7 @@ export class ApexActionController {
     let context;
     let name;
     const telemetryService = await getTelemetryService();
+    const isESRDecomposed = await this.isESRDecomposed();
```

Comment:
@daphne-sfdc make it a field in the class, then you will not have to change method signatures to pass the value around.